airbyte_cdk/sources/declarative/decoders/json_decoder.py

```diff
@@ -10,47 +10,39 @@
 import orjson
 import requests
 
+from airbyte_cdk.sources.declarative.decoders import CompositeRawDecoder, JsonParser
 from airbyte_cdk.sources.declarative.decoders.decoder import Decoder
 
 logger = logging.getLogger("airbyte")
 
 
-@dataclass
 class JsonDecoder(Decoder):
     """
     Decoder strategy that returns the json-encoded content of a response, if any.
     """
 
-    parameters: InitVar[Mapping[str, Any]]
+    def __init__(self, parameters: Mapping[str, Any]):
```

Comment:
Nit: a comment on what `_decoder` is would be really good to have a few months form now.

Reply:
Added a description in the class documentation